src/diffusers/modeling_flax_utils.py

```diff
@@ -394,6 +393,76 @@ def from_pretrained(
         # flatten dicts
         state = flatten_dict(state)
 
+        prng_key = jax.random.PRNGKey(0)
+        params_shape_tree = jax.eval_shape(model.init_weights, prng_key)
```

Comment:
If this is not used anywhere down in the code
```suggestion
        params_shape_tree = jax.eval_shape(model.init_weights, rng=jax.random.PRNGKey(0))
```